README.md

```diff
@@ -31,26 +43,29 @@ const encrypted = await evervaultClient.encrypt({ ssn: '012-34-5678' });
 const result = await evervaultClient.run('<CAGE_NAME>', encrypted);
 ```
 
-## API Reference
+## Reference
+
+At present, there are three functions available in the Node.js SDK: `evervault.encrypt()`,`evervault.run()`, and `evervault.cagify`.
```

Comment:
```suggestion
```

Reply:
went with: "The Evervault Node.js SDK exposes three functions."